src/tutorials/Mutable-File-System/03-exercise.md

```diff
@@ -1,3 +1 @@
-Let's add some files to IPFS using MFS. Since `files` is an array representing the files currently available to us in the browser, we'll need to loop through the array and use `files.write()` to add each file we find there to IPFS, one at a time. (Only have a single file uploaded? No problem!)
-
-Put your files in the root directory (`/`) and be sure to add the name of each file to the path when you add it (**hint:** the file object in the browser stores the filename as `file.name`). Be sure to set up your options so that a new file is created when one isn't found at the given path.
+First, upload one or more files by dragging and dropping below or clicking to make a selection from your file explorer.  Next, in the code editor, remove the comment markers (`//`) that precede `return files` within the `run` function.
```

Comment:
<img width="720" alt="Screenshot 2019-05-31 at 14 40 34" src="https://user-images.githubusercontent.com/152863/58709517-3f3eac00-83b2-11e9-99f8-2f45cd920f10.png">

Multiple file uploads do not grow the container to accomodate

Reply:
Fixed, thanks!